example/lib/peripheral_list/components/rssi_view.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:blemulator_example/styles/custom_text_style.dart';
+import 'package:blemulator_example/util/signal_level.dart';
+import 'package:flutter/material.dart';
+
+class RssiView extends StatelessWidget {
+  final int _rssi;
+
+  RssiView(this._rssi);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.end,
+      children: <Widget>[
+        Text(
+          _formatRssi(),
+          style: CustomTextStyle.cardValueAccessory
+              .copyWith(color: signalLevelForRssi(_rssi).color()),
+        ),
+        Padding(
+          padding: const EdgeInsets.only(left: 4.0),
+          child: Icon(
+            Icons.settings_input_antenna,
+            color: signalLevelForRssi(_rssi).color(),
```

Comment:
If `color()` extension is and will be used only here, it should be in this file and private.

Reply:
It's almost certainly going to be used on the PeripheralDetailsScreen as well.

Reply:
Isn't the point here to reuse the widget, not just the colour?

Reply:
Well, if we look at it this way then yes, the extension could live here in this file as a private extension.

What about `SignalLevel` then - it's probably going to be used only by this widget as well, should it be moved to this file as private?

Reply:
I don't think so, since it's the logic deciding whether the signal is strong. WDYT @pawelByszewski?

Reply:
As I understand `SignalLevel` is part of the logic/model whereas picking color is the responsibility of View

Reply:
Moved the extension to the rssi_view.dart and made it private after further discussion.